Cargo.toml

```diff
@@ -50,7 +50,7 @@ futures-executor = { version = "0.3.5", default-features = false }
 futures-io = { version = "0.3.5", default-features = false }
 futures-util = { version = "0.3.5", default-features = false }
 async-std = "1.6"
-tokio = "1.21"
+tokio = "= 1.38.1"
```

Comment:
Pinning dependencies like this is generally a bad idea, since this also requires all downstream consumers of our crates to use tokio 1.38.1. Instead, the `Cargo.lock` file is the proper approach to pinning dependencies for the purpose of CI, but that's not working here for some reason?